exercises/en/test_01_09a.py

```diff
@@ -8,6 +8,11 @@ def test():
     # Since we haven't started assigning charts to variable names yet,
     # this might be the better way to test for the first exercise.
     # Maybe even for later exercises.
+    assert 'from vega_datasets import data' in __solution__, "Did you import the correct library for data from vega_datasets?"
+    assert 'data.stocks()' in __solution__, "Make sure to call the stocks data from the data library"
+    assert '# alt.Chart' not in __solution__, "Make sure you remove the hash (#) symbol used to comment out the plotting codes."
+    assert 'alt.Chart(stocks)' in __solution__, "Make sure you're passing the correct data object fot plotting."
+    assert 'mark_line' in __solution__, "Make sure you're using the correct type of mark"
     assert 'x="date"' in __solution__, "Did you encode the x channel correctly?"
     assert 'y="price"' in __solution__, "Did you encode the y channel correctly?"
     assert 'color="symbol"' in __solution__, "Did you encode the color channel correctly?"
```

Comment:
I'm curious if we can make these more robus by checking x="date" OR `x= "date"` to account for the white space issue? (for x, y, and color)

Reply:
For this plot in `01_09a`, I couldn't figure out a way to do it, but I've modified the test for `01_09b` to check that the attributes for the plot object that was given in the sample code remain unmodified (as a practice for future exercises where students will bind names to the plots )